desktop.patch

```diff
@@ -0,0 +1,46 @@
+From 93af955a2976330cbdb855f1491500653c534152 Mon Sep 17 00:00:00 2001
```

Comment:
you could also, instead of a patch, use `desktop-file-edit`.
not blocking anything though.